src/test/java/uiStepDefinitions/AddEducation.java

```diff
@@ -0,0 +1,35 @@
+package uiStepDefinitions;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Map;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import utilities.DriverFactory;
+
+public class AddEducation {
+
+	private WebDriver driver = DriverFactory.getInstance();
+
+	@Then("User updates {string}")
+	public void user_updates(String string, io.cucumber.datatable.DataTable dataTable) throws InterruptedException {
```

Comment:
- The step doesn't make sense, it should say something like `user adds an education credential`
- There should be just 1 data table as a parameter, not sure about that String